packages/web/src/pages/pay-and-earn-page/components/YourCoins.tsx

```diff
@@ -78,58 +87,76 @@ export const YourCoins = () => {
     isAudioPriceLoading
   } = useFormattedAudioBalance()
 
+  const { data: bonkBalance, status: bonkBalanceStatus } = useTokenBalance({
```

Comment:
it is probably best to move these into card specific sections cause as we add more itll get quite busy and potentially non-performant having all this at the root, so perhaps we have "AudioCoinCard" and "BonkCoinCard"?